src/diff.js

```diff
@@ -29,6 +29,10 @@
  *
  * QUnit.diff( "the quick brown fox jumped over", "the quick fox jumps over" ) === "the  quick <del>brown </del> fox jump<ins>s</ins><del>ed</del over"
  */
+
+// Don't load QUnit.diff on non-Browser environments
+if ( typeof window !== "undefined" ) {
```

Comment:
when QUnit is not exposed as a global value anymore, the following QUnit.difff line would fail.